assets/global.js

```diff
@@ -529,32 +530,51 @@ class SliderComponent extends HTMLElement {
     this.update();
   }
 
+  resetPages() {
+    this.sliderItems = this.slider.querySelectorAll('li');
```

Comment:
This seem specific to sliders using `li` elements. Should it target the elements differently so it's a bit more generic and could be used in any slider situation?

Reply:
Good catch. Forgot to update after rebase.